lib/galaxy/webapps/uwsgi_schema.yml

```diff
@@ -966,6 +966,7 @@ mapping:
   static-map: {desc: ' map mountpoint to static directory (or file)', type: any}
   static-map2: {desc: ' like static-map but completely appending the requested resource
       to the docroot', type: any}
+  static-map3: {desc: ' like static-map but only mapping favicon to bare root', type: any}
```

Comment:
`static-map2` is an actual uWSGI option, you shouldn't need to modify this.

Reply:
removed, thanks